frontend/ui/widget/pagebrowserwidget.lua

```diff
@@ -1275,6 +1406,46 @@ function PageBrowserWidget:onScrollRowDown()
     return true
 end
 
+function PageBrowserWidget:_wrapAroundY(dy)
```

Comment:
May be add a comment above, something like:
`-- Override FocusManager internal methods, so we can scroll the view instead of wrap around`